03_Run_Locally.ipynb

```diff
@@ -52,7 +52,7 @@
    },
    "outputs": [],
    "source": [
-    "selected_subscription=\"YOUR_SUBSCRIPTION\"\n",
+    "subscription_id=\"YOUR_SUBSCRIPTION\"\n",
```

Comment:
I know you're trying to regularize across happy paths, but this change is for ease in computer manipulation at the expense of human undesirability. I don't think you should do this. The happy paths were made for humans to use.